Design-Docs/tools/scs-compatible.yaml

```diff
@@ -6,11 +6,13 @@ iaas:
     standards:
       - name: Flavor naming
         url: https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Standards/SCS-0003-v2-flavor-naming.md
-        check_tool: flavor-names-openstack.py
+        check_tools:
+          - url: flavor-names-openstack.py
```

Comment:
_url_ is somewhat misleading if you are using local files like here. Maybe rather _uri_ or _source_?

Reply:
Good point. `executable` would seem to best describe what this is. Something that can be run...
Opinions?

Reply:
This!